webservice/model/user.go

```diff
@@ -107,28 +107,28 @@ func UpdateUserEmail(userID int, email string) bool {
 }
 
 //UpdateUserPassword updates the users password in the db
-func UpdateUserPassword(userID int, password string) bool {
+func UpdateUserPassword(userID int, password string) error {
 
 	// Hash the password first
 	pass, err := hashPassword(password)
 
 	if err != nil {
 		//todo log error
 		log.Fatal(err.Error())
```

Comment:
log.Fatal will stop application.
No need to log error inside function, as function returns an error